src/components/Contacts/ContactDetails/ContactDetails.tsx

```diff
@@ -0,0 +1,43 @@
+import { AppBar, Box, Tab, Tabs } from '@material-ui/core';
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useGetContactDetailsLazyQuery } from './ContactDetails.generated';
+import { ContactDetailsHeader } from './ContactDetailsHeader/ContactDetailsHeader';
+
+interface Props {
+  accountListId: string;
+  contactId: string | null;
```

Comment:
I don't think this should be nullable. The parent page should render this component if there is no contact to show.